ui/src/app/pages/apps-routes/release-notes/release-notes.page.ts

```diff
@@ -0,0 +1,53 @@
+import { Component } from '@angular/core'
+import { ActivatedRoute } from '@angular/router'
+import { ApiService } from 'src/app/services/api/api.service'
+import { ReleaseNoteModel } from './release-notes.model'
+
+@Component({
+  selector: 'release-notes',
+  templateUrl: './release-notes.page.html',
+  styleUrls: ['./release-notes.page.scss'],
+})
+export class ReleaseNotes {
+  error = ''
+  pkgId: string
+  releaseNotes: { [version: string]: string}
```

Comment:
Probably better to just read the release notes right from the release notes model. Why even have the local variable. Then, getReleaseNotes() could just set the value in the model.